code/obj/item/clothing/suits.dm

```diff
@@ -2146,14 +2149,24 @@ ABSTRACT_TYPE(/obj/item/clothing/suit/sweater_vest)
 	desc = "A piece of soggy notebook paper with a red S doodled on it, presumably to represent security."
 	icon_state = "security_badge_paper"
 
-/obj/item/clothing/suit/hosmedal
+/obj/item/clothing/suit/security_badge/hosmedal
 	name = "war medal"
 	desc = ""
 	icon = 'icons/obj/clothing/overcoats/item_suit_gimmick.dmi'
 	w_class = W_CLASS_TINY
 	wear_image_icon = 'icons/mob/clothing/overcoats/worn_suit_gimmick.dmi'
 	icon_state = "hosmedal"
 	icon_state = "hosmedal"
+	var/award_text = "This is a medal. There are many like it, but this one's mine."
```

Comment:
This appears to everyone when used by anyone to anyone so I don't think this is great flavour text, it should probably say something like "Awarded for great bravery." because its supposed to be a war medal, though im not sure the flavor text is needed.

Reply:
This actually gets overwritten by the `/framed_award/hos_medal` it spawns in. The only way you'd ever see that `award_text` is if it was hand-spawned by admins.